package-lock.json

```diff
@@ -2555,6 +2555,12 @@
         "z-schema": "~5.0.2"
       }
     },
+    "node_modules/@rushstack/node-core-library/node_modules/@types/node": {
```

Comment:
no `package-lock.json` are required as `package.json` is untouched, rebase issue...